### PR TITLE
274 minimize archive

### PR DESCRIPTION
**Pull request by SepplL**
Refers to #274 and closes issue.

tarball all old directories as discussed in meeting.

### CODE REVIEW

**Review by aknierim — CHANGES_REQUESTED** [review]
We should remove the old (i.e. now dead) links from `archive.md`. Besides that, LGTM.

**Comment by SepplL** [comment]
> We should remove the old (i.e. now dead) links from `archive.md`. Besides that, LGTM.

I was working on exactly that in the moment you commented.
I plainly linked against the tarball every time to preserve the current timestamps of editing and the daily progress.

**Comment by aknierim** [comment]
Wouldn't it make more sense to just link every tarball once? Something like this (keeping the last year as is, then adding links to the tarballs):

## Wintersemester 2023/24
### Toolbox
...
### LaTeX
...

## Wintersemester 2022/23
* [Archive download] (2023.tar.gz)

## Wintersemester 2021/22
* [Archive download] (2022.tar.gz)
...

**Review by aknierim — APPROVED** [review]
LGTM!

**Comment by chrbeckm** [comment]
Do we upload the files from the running workshop into a `current` dir then?

**Comment by SepplL** [comment]
Or we initialize a new year and set the `latest` link after the workshop is finished. Then keep two years for some time and later move to only storing the tarball.
We could think of a schedule when to change the archive. A couple of months after the workshop is done maybe?

**Comment by SepplL** [comment]
Now updated to include proper `current` directory with all links as discussed in the last meeting.
This should be the last change before adding the files for this year (if approved ofc)

File: archive.md [hunk 1]
```diff
@@ -5,390 +5,123 @@ title: Archiv
...
-- [Aufgaben Tag 3](/files/archive/2023/exercises-latex-3.zip) (Stand 04.10.2023)
-- [Aufgaben Tag 4](/files/archive/2023/exercises-latex-4.zip) (Stand 05.10.2023)
+- [Folien](/files/archive/current/latex.pdf){:target="_blank"} (Stand 06.10.2023)
+- [Aufgaben Tag 1](/files/archive/current/exercises-latex-1.zip) (Stand 02.10.2023)
+- [Aufgaben Tag 2](/files/archive/current/exercises-latex-2.zip) (Stand 04.10.2023)
+- [Aufgaben Tag 3](/files/archive/current/exercises-latex-3.zip) (Stand 04.10.2023)
+- [Aufgaben Tag 4](/files/archive/current/exercises-latex-4.zip) (Stand 05.10.2023)
 - <a href="https://github.com/pep-dortmund/toolbox-workshop-protocol-template" target="_blank">LaTeX Vorlage für Protokolle</a>
 
+#### Alles zusammen als Archive
```
Comment (aknierim) [inline archive.md:42]: Maybe change "Archive" to German singular "Archiv"? This sounds somewhat off otherwise

**Review by aknierim — CHANGES_REQUESTED** [review]
Just a minor change and we can merge :)

**Comment by SepplL** [comment]
Done. Renamed all `Archive` to `Archiv`.
This begs the question whether we want to offer the archives as more accessible `zip` files at some point as well?
This is no necessary change as of now, but if multiple people ask about this, it could be a viable change later down the road.

**Comment by aknierim** [comment]
> This begs the question whether we want to offer the archives as more accessible `zip` files at some point as well? This is no necessary change as of now, but if multiple people ask about this, it could be a viable change later down the road.

Thought about that too. I guess this is definitely something worth considering for the future.

**Review by aknierim — APPROVED** [review]
Looks good :)

**Comment by SepplL** [comment]
Now edited the material section to also utilize the `current` link structure as well. If there are no complaints, I would merge the current state later.

**Review by aknierim — APPROVED** [review]

**Pull request merged**